Character.cpp

```diff
@@ -98,20 +98,32 @@ void Character::attackInternal(Character& other)
             b) your stats are boosted 10%
             c) the initial value of your stats is updated to reflect this boosted stat for the next time you defeat another character.
       */
-        assert(false);
+        hitPoints = *initialHitPoints;
+        armor = *initialArmorLevel;
+        attackDamage = *initialAttackDamage;
+
+        boostArmor(armor * 0.1);
+        boostHitPoints(hitPoints * 0.1);
+        boostAttackDamage(attackDamage * 0.1);
+
+        *initialHitPoints = hitPoints;
+        *initialArmorLevel = armor;
+        *initialAttackDamage = attackDamage;
+
```

Comment:
I see. I put the process into a lvlUpStat function, then put the 3 calls of that into a lvpUpChar function. Thought if characters had to be leveled up in different ways it could be useful?